src/main/java/duke/commands/EventCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.commands;
+
+import java.time.LocalDate;
+
+import duke.storage.Storage;
+import duke.tasks.Event;
+import duke.tasks.TaskList;
+import duke.ui.UI;
+
+/**
+ * Represents an Event command when the user wants to add
+ * a new Event.
+ */
+public class EventCommand extends Command {
+    public static final String COMMAND_WORD = "event";
+    private String eventDate;
+    private LocalDate eventLocalDate;
+    private boolean hasLocalDate;
```

Comment:
Perhaps you could rename it to "isLocalDateFormatted" instead, since the Date is definitely present, is just a matter if it is formatted correctly or not. Just a suggestion!! You do what you think is right 😺

Reply:
Its a slight difference but I think you're right, naming it the way you suggested definitely conveys what the boolean is for more effectively 😃